src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -119,6 +132,79 @@ class ParticipantProjectSpeciesStore(
         .execute()
   }
 
+  fun fetchParticipantProjectsForSpecies(
+      organizationId: OrganizationId,
+      speciesId: SpeciesId
+  ): List<ParticipantProjectsForSpecies> {
+    val today = LocalDate.ofInstant(clock.instant(), TimeZones.UTC)
+    val user = currentUser()
+
+    return dslContext
+        .select(
+            DELIVERABLES.ID,
+            PROJECTS.NAME,
+            PROJECTS.ID,
+            PARTICIPANT_PROJECT_SPECIES.ID,
+            PARTICIPANT_PROJECT_SPECIES.SUBMISSION_STATUS_ID,
+            SPECIES.ID)
+        .from(SPECIES)
+        .join(PARTICIPANT_PROJECT_SPECIES)
+        .on(SPECIES.ID.eq(PARTICIPANT_PROJECT_SPECIES.SPECIES_ID))
+        .join(PROJECTS)
+        .on(PARTICIPANT_PROJECT_SPECIES.PROJECT_ID.eq(PROJECTS.ID))
+        .join(ORGANIZATIONS)
+        .on(PROJECTS.ORGANIZATION_ID.eq(ORGANIZATIONS.ID))
```

Comment:
Is joining with `organizations` necessary here?

As I mentioned in an earlier comment, the organization is implied by the species, so this function doesn't need to take an organization ID argument at all. But if it were to take one, `.where(PROJECTS.ORGANIZATION_ID.eq(organizationId))` would work without this join.

Reply:
No need for the join, thanks